src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java

```diff
@@ -135,6 +135,26 @@ public HttpResponse getAuthInfo(Header... headers) {
         return executeRequest(new HttpGet(getHttpServerUri() + "/_opendistro/_security/authinfo?pretty"), headers);
     }
 
+    public HttpResponse getOBOTokenFromOboEndpoint(String jsonData, Header... headers) {
```

Comment:
nit: Change name to `getOnBehalfOfToken`? Everything in this class calls on an endpoint so it may be redundant to have it in the method name.

Reply:
Fixed